ARTICLE-1.markdown

```diff
@@ -95,13 +95,7 @@ stated above, upon consensus.
 
 ### Section 1.3  Acceptance of Member-Owners
 
-(a)	Any subset of the Member Body so inclined, or an
-empowered hiring committee appointed by the interested subset
-of the Member Body shall review applications, instate a period
-of member candidacy, and once criteria stated above are
-established or ignored by the interested subset of the Member
-Body, shall give an offer of Member-Ownership to the potential
-Member-Owner.
+(a)	Empowered comittee appointed by guiding council (aka The Board, see Section 3.4) shall give an offer of Member-Ownership to the potential Member-Owner once criteria stated have been met.
```

Comment:
The change has been made to make clear that the committee giving offer of membership to the new member owner has to be appointed by the board. The earlier wording was not clear in this aspect.